### ISSUE TITLE
[A11Y] OJS Reader View - Redundant and unlabeled links on article images, and titles

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12668]
## Issue Description  
On the  Current Issue grid, multiple links are provided for the same article. Specifically, the article image, and the title  creating separate interactive targets.

This results in redundant links that increase navigation effort for keyboard and screen reader users, as users must tab through multiple links that may lead to the same or closely related destinations.

Additionally, article images are currently given empty `alt` attributes (null alt), which is appropriate for decorative images. However, because these images are also interactive links, they lack an accessible name, which may cause confusion for users who rely on screen readers.

## How to reproduce / Current behavior  
1. Navigate to an OJS home page displaying Current Issue with image/cover.  
2. Use the `Tab` key to navigate through article listings.  
3. Observe:
   - The article image is a link.  
   - The article title is also the same link.  
4. Inspect the image link and note that it has a null (`alt=""`) attribute and no accessible name.  

## What application are you using?  
OJS  version 3.5.0-3  

## Stack used  
- Device: Desktop  
- OS: Windows/macOS  
- Browser: Chrome  

## Testing method  
- Method: Manual (Keyboard and Screen Reader)  

## Action  
- Remove the link (`href`) from article images so that only the article title serves as the primary navigation link.  
- Alternatively, wrap the image and title within a single `<a>` element so they function as one unified link.  
- Ensure that:
  - Images used as links have appropriate `alt` text describing their purpose, **or**
  - Images remain decorative (`alt=""`) only if they are not interactive.  
 

### Screenshot

<img width="880" height="255" alt="image and title using same link" src="https://github.com/user-attachments/assets/40485be9-89b5-4aff-b0b1-6a0be0d058f8" />

### HTML
Chunk of code showing empty `ALT` attribute
```
<a href="https://ojs35.demo.publicknowledgeproject.org/index.php/demojournal/en/article/view/6" class="file">
								<img loading="lazy" src="https://ojs35.demo.publicknowledgeproject.org/public/journals/1/submission_6_6_coverImage_en_US.jpg" alt="">
			</a>
```

### WCAG Success Criteria  
2.4.4 Link Purpose (A)  
2.4.3 Focus Order (A)  
1.1.1 Non-text Content (A)
